packages/web-react/src/types/index.ts

```diff
@@ -11,6 +11,7 @@ export * from './heading';
 export * from './icon';
 export * from './label';
 export * from './link';
+export * from './main';
```

Comment:
Please, recheck this. But I think that the content of `main.ts` is unused and therefor we should delete it.